app/settings/surveys/targeted-surveys/targeted-survey-edit.html

```diff
@@ -180,9 +188,28 @@ <h2 class="stepper-heading">
                                 <span class="stepper-badge">4</span>
                                 <span translate="survey.targeted_survey.publish"></span>
                             </h2>
+                            <div class="alert" ng-show="isActiveStep(4)">
+                                <p translate="survey.targeted_survey.publish_description" translate-values="{questions: targetedSurvey.stepThree.questions.length, numbers:finalNumbers.goodNumbers.length, sms: sms, cost:cost}"></p>
```

Comment:
So technically there should be 4 possible text options:
- option 1: 1 questions to 1 person
- option 2: 1 questions to 2+ people
- option 3: 2+ questions to 1 person
- option 4: 2+ questions to 2+ people

I don't know any other way around this to make sure that the translations are correct.

Reply:
You can see similar issues in my latest PR where I had to put in multiple translation options. https://github.com/ushahidi/platform-client/pull/1036

Reply:
Good catch, fixed now!